sdk/extensions/wcf/Microsoft.CoreWCF.Azure.StorageQueues/tests/Microsoft.CoreWCF.Azure.StorageQueues.Tests.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <GenerateRuntimeConfigDevFile>true</GenerateRuntimeConfigDevFile>
+    <IsPackable>false</IsPackable>
+    <UserSecretsId>7d774a2c-019d-4f91-87ed-06aeedf5fd0d</UserSecretsId>
```

Comment:
Never seen this element before.   What's that used for and is it considered sensitive?

Reply:
I presume it's safe to remove this as we're getting all our credentials from the azurite fixture code?

Reply:
Yes, please remove